app-antivirus/clamtk/clamtk-5.25.ebuild

```diff
@@ -0,0 +1,170 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PLOCALES="af ar ast az bg bs ca cs da de el_GR en_AU en_CA en_GB eo es eu fi fo fr ga gl he hr hu id it ja km ko lo lt lv mr ms nb nl nl_BE nn pa pl pt pt_BR ro ru si sk sl sr@latin sv ta te th tr ug uk uz zh_CN zh_TW"
+PLOCALE_BACKUP="en_GB"
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit desktop l10n perl-functions python-single-r1 xdg-utils
+
+MY_PV_GNOME="0.02"
+MY_PV_KDE="0.18"
+MY_PV_MATE="0.04"
+MY_PV_XFCE="0.06"
+
+DESCRIPTION="A graphical front-end for ClamAV"
+HOMEPAGE="https://dave-theunsub.github.io/clamtk/"
+SRC_URI="https://bitbucket.org/davem_/clamtk/downloads/${P}.tar.xz
+	gnome? ( https://bitbucket.org/davem_/clamtk-gnome/downloads/clamtk-gnome-${MY_PV_GNOME}.tar.xz )
+	kde? ( https://bitbucket.org/davem_/clamtk-kde/downloads/clamtk-kde-${MY_PV_KDE}.tar.xz )
+	mate? ( https://bitbucket.org/davem_/nemo-sendto-clamtk/downloads/nemo-sendto-clamtk-${MY_PV_MATE}.tar.xz )
```

Comment:
nemo is cinnamon's file manager, not mate's. are those compatible? =)

Reply:
Whoops. No clue how I mixed this up. Changed now all `mate` to `cinnamon`. I don't know, if they are compatible.